libs/ledger-live-common/src/__tests__/test-helpers/bridgeTeardownUpdateMocks.ts

```diff
@@ -0,0 +1,59 @@
+/* eslint-disable no-console */
+import axios from "axios";
+import { StdRequest } from "./types";
+import { promises as fsp } from "fs";
+
+/* Creates a mock file from registered request */
+async function createMock(stdRequest: StdRequest, path: string) {
+  // usual instance has errors managed by live-network lib
+  const uninterceptedAxiosInstance = axios.create();
+
+  const createMockOp = async resp => {
+    const responseInfo = { ...stdRequest.response };
+    // Stores headers etc but no data
+    delete stdRequest.response;
```

Comment:
Why do you mutate `stdRequest`?

Reply:
Nice catch ! It was used before but not anymore, thanks